torq.q

```diff
@@ -596,12 +595,10 @@ if[not `noconfig in key .proc.params;
 .proc.loadprocesscode:@[value;`.proc.loadprocesscode;1b];
 .proc.loadnamecode:@[value;`.proc.loadnamecode;0b];
 .proc.loadhandlers:@[value;`.proc.loadhandlers;1b];
-.proc.logroll:@[value;`.proc.logroll;1]
-.lg.o[`init;".proc.loadcommoncode flag set to ",string .proc.loadcommoncode];
-.lg.o[`init;".proc.loadprocesscode flag set to ",string .proc.loadprocesscode];
-.lg.o[`init;".proc.loadnamecode flag set to ",string .proc.loadnamecode];
-.lg.o[`init;".proc.loadhandlers flag set to ",string .proc.loadhandlers];
-.lg.o[`init;".proc.logroll flag set to ",string .proc.logroll];
+.proc.logroll:@[value;`.proc.logroll;1b]
```

Comment:
is this definitely meant to be a boolean? (it probably is, but just in case it was intentionally a long instead)

also while we're fixing it

```suggestion
.proc.logroll:@[value;`.proc.logroll;1b];
```

Reply:
every other instance of it being assigned seems to be with a boolean and its currently set to 1b so I assumed its just a typo, was kind of hoping you would have confirmed haha, any ideas how to be more sure than this?

Reply:
if it's a boolean everywhere else, I think that's fine 👍